mephisto/tools/data_browser.py

```diff
@@ -89,3 +90,15 @@ def get_data_from_unit(self, unit: Unit) -> Dict[str, Any]:
             "task_start": agent.state.get_task_start(),
             "task_end": agent.state.get_task_end(),
         }
+
+    def get_workers_with_qualification(self, qualification_name: str) -> List[Worker]:
+        """
+        Returns a list of 'Worker's for workers who are qualified wrt `qualification_name`.
+        """
+        qual_list = self.db.find_qualifications(qualification_name=qualification_name)
+        assert len(qual_list) >= 1, f"No qualification found named {qualification_name}"
+        qualification_id = qual_list[0].db_id
+        qualifieds = self.db.check_granted_qualifications(
+            qualification_id=qualification_id, value=1
+        )
+        return [Worker.get(self.db, qual.worker_id) for qual in qualifieds]
```

Comment:
I agree. It is better to get the worker object and then decide what you want to do with it 👍